### DISCUSSION TITLE
QCefView有类似QWebEngineView那样对单个网页进行设置的类吗

### DISCUSSION THREAD

**Discussion by sslcui** [CefView/QCefView#344] [Q&A]
就像QWebEngineView中的QWebEnginePage类

**Comment by L-Super** [comment]
See：https://cefview.github.io/QCefView/zh/docs/reference/QCefContext

**Reply by tishion** [reply]
应该是这个：
https://cefview.github.io/QCefView/zh/docs/reference/QCefSetting

QCefView的构造函数支持传入QCefSetting，这个QCefSetting只对这个实例生效。
